editor/editor_help.cpp

```diff
@@ -363,7 +363,7 @@ void EditorHelpIndex::_tree_item_selected() {
 		return;
 
 	emit_signal("open_class", s->get_text(0));
-
+	EditorNode::get_singleton()->call("_editor_select", EditorNode::EDITOR_SCRIPT);
```

Comment:
While at it, maybe you could replace the ` EditorNode *editor` in `select_class` to use the singleton directly? So that the style is consistent in both functions.